Index: src/main/java/com/testfabrik/webmate/javasdk/browsersession/BrowserSessionScreenshotExtractionConfig.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/testfabrik/webmate/javasdk/browsersession/BrowserSessionScreenshotExtractionConfig.java	(revision c8f22169f3598803ab3bedd6d61549cb104e7749)
+++ src/main/java/com/testfabrik/webmate/javasdk/browsersession/BrowserSessionScreenshotExtractionConfig.java	(date 1575015578000)
@@ -10,6 +10,8 @@
 
     private boolean hideFixedElements;
 
+    private boolean useTranslateYScrollStrategy;
+
     public boolean isHideFixedElements() {
         return hideFixedElements;
     }
@@ -26,6 +28,12 @@
         this.fullPage = fullPage;
     }
 
+    public void setUseTranslateYScrollStrategy(boolean useTranslateYScrollStrategy) {
+        this.useTranslateYScrollStrategy = useTranslateYScrollStrategy;
+    }
+
+    public boolean isUseTranslateYScrollStrategy(){ return this.useTranslateYScrollStrategy;}
+
     /**
      * Creates a new BrowserSession ScreenshotExtractionConfig instance.
      * @param fullPage Whether fullpage screenshots should be taken or not.
@@ -34,5 +42,12 @@
     public BrowserSessionScreenshotExtractionConfig(boolean fullPage, boolean hideFixedElements) {
         this.fullPage = fullPage;
         this.hideFixedElements = hideFixedElements;
+        this.useTranslateYScrollStrategy = false;
+    }
+
+    public BrowserSessionScreenshotExtractionConfig(boolean fullPage, boolean hideFixedElements, boolean useTranslateYScrollStrategy) {
+        this.fullPage = fullPage;
+        this.hideFixedElements = hideFixedElements;
+        this.useTranslateYScrollStrategy = useTranslateYScrollStrategy;
     }
 }
